schema/config-2.0.json

```diff
@@ -23,9 +23,6 @@
         "roots": {
             "type": "object"
         },
-        "imageio": {
-            "type": "object"
-        },
```

Comment:
Can we just remove this? Are old projects ever checked against this schema and invalidated if it finds an entry that isn't in this config?

It's not a required key so that helps, but still.